recipes/antlr-cpp-runtime/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "antlr-cpp-runtime" %}
+{% set version = "4.12.0" %}
+{% set sha256 = "642d59854ddc0cebb5b23b2233ad0a8723eef20e66ef78b5b898d0a67556893b" %}
```

Comment:
except for `version`, it's usually simpler to avoid single-use variables (e.g. `sha`, `name`), leaving them in-place

Reply:
I agree, but this is the pattern that we've been using for a lot of these recipes ive been adding

Reply:
The bot support this for now but future iterations or code clean-ups may remove it. We don't need jinja variables that are used only once and adds more complexity than simplification. I recommend just pasting the checksum in the field below.

Reply:
@timkpaine do you want to address this one before we merge.

Reply:
done